internal/bytesconv/errors.go

```diff
@@ -41,13 +41,13 @@
 
 package bytesconv
 
-import "errors"
+import "github.com/cloudwego/hertz/pkg/common/errors"
 
 var (
-	errEmptyInt               = errors.New("empty integer")
-	errUnexpectedFirstChar    = errors.New("unexpected first char found. Expecting 0-9")
-	errUnexpectedTrailingChar = errors.New("unexpected trailing char found. Expecting 0-9")
-	errTooLongInt             = errors.New("too long int")
-	errEmptyHexNum            = errors.New("empty hex number")
-	errTooLargeHexNum         = errors.New("too large hex number")
+	errEmptyInt               = errors.NewPublic("empty integer")
```

Comment:
No need to change this - errors defined here act as core error, which is for cross-package.

Reply:
done, thx